glide.lock

```diff
@@ -96,20 +119,110 @@ imports:
   - status
   - tap
   - transport
+- name: gopkg.in/inf.v0
+  version: 3887ee99ecf07df5b447e9b00d9c0b2adaa9f3e4
 - name: gopkg.in/yaml.v2
-  version: eb3733d160e74a9c7e442f435eb3bea458e1d19f
+  version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 - name: k8s.io/api
-  version: edd5e319fc45211023fffb41c7aa5555229c4179
+  version: 0f11257a8a25954878633ebdc9841c67d8f83bdb
+  subpackages:
+  - admissionregistration/v1alpha1
+  - admissionregistration/v1beta1
+  - apps/v1
+  - apps/v1beta1
+  - apps/v1beta2
+  - authentication/v1
+  - authentication/v1beta1
+  - authorization/v1
+  - authorization/v1beta1
+  - autoscaling/v1
+  - autoscaling/v2beta1
+  - batch/v1
+  - batch/v1beta1
+  - batch/v2alpha1
+  - certificates/v1beta1
+  - core/v1
+  - events/v1beta1
+  - extensions/v1beta1
+  - networking/v1
+  - policy/v1beta1
+  - rbac/v1
+  - rbac/v1alpha1
+  - rbac/v1beta1
+  - scheduling/v1alpha1
+  - settings/v1alpha1
+  - storage/v1
+  - storage/v1alpha1
+  - storage/v1beta1
+- name: k8s.io/apiextensions-apiserver
+  version: 898b0eda132e1aeac43a459785144ee4bf9b0a2e
+  subpackages:
+  - pkg/features
 - name: k8s.io/apimachinery
-  version: 01bc873149a1802eb74df583613872d126449ed5
+  version: f6313580a4d36c7c74a3d845dda6e116642c4f90
```

Comment:
After running `glide update`, I've manually updated this to the version that is also seen in [helm's glide.lock](https://github.com/helm/helm/blob/cc7cc1087f586f64d0b0d08aa516a003ced2d85b/glide.lock#L401).
From my observation, it was required in order to fix build errors due to version mismatch between apimachinery and [json-iterator](https://github.com/databus23/helm-diff/pull/82/files#diff-f16a80eae23d5b298c2652448ec420cfR45).

Reply:
According to the discussion on the k8s slack, I'm going to add respective `version: ...` settings to affected dependencies, so that we won't need to manually update these. Thx for your review @databus23!